Makefile

```diff
@@ -159,10 +161,17 @@ test-coverage:
 	@export VERSION=$(VERSION); bash -x scripts/test_cover.sh
 .PHONY: test-coverage
 
+## test-fuzz: Run all fuzz tests.
 test-fuzz:
 	bash -x scripts/test_fuzz.sh
 .PHONY: test-fuzz
 
+## test-interchain: Run interchain tests in verbose mode. Requires Docker.
+test-interchain:
+	@echo "--> Running interchain tests"
+	@go test ./test/interchain -v
+.PHONY: test-interchain
```

Comment:
Can we add this to the CI then

Reply:
I'm hesitant to add it to CI now because this test currently uses a static docker image https://github.com/rootulp/celestia-app/blob/08ca4d2fe7971d69b71a39a1e2c91dbdb50e72ce/test/interchain/chainspec/celestia.go#L16

so even if a PR breaks ICA, we wouldn't get signal by running it in CI unless we update that static docker image to use a Docker image that is created on a per PR basis. Will create a follow up issue to add it to CI

Reply:
Created https://github.com/celestiaorg/celestia-app/issues/3358